Sources/SignalRClient/HubConnection.swift

```diff
@@ -90,20 +94,35 @@ public actor HubConnection {
     }
 
     public func send(method: String, arguments: Any...) async throws {
-        // Send a message
+        let invocationMessage = InvocationMessage(target: method, arguments: AnyEncodableArray(arguments), streamIds: nil, headers: nil, invocationId: nil)
+        let data = try hubProtocol.writeMessage(message: invocationMessage)
+        try await sendMessageInternal(data)
+    }
+
+    public func invoke(method: String, arguments: Any...) async throws -> Void {
+        let (invocationId, tcs) = await invocationHandler.create()
+        let invocationMessage = InvocationMessage(target: method, arguments: AnyEncodableArray(arguments), streamIds: nil, headers: nil, invocationId: invocationId)
+        let data = try hubProtocol.writeMessage(message: invocationMessage)
+        try await sendMessageInternal(data)
+        _ = try await tcs.task()
     }
 
-    public func invoke(method: String, arguments: Any...) async throws -> Any {
-        // Invoke a method
-        return ""
+    public func invoke<TReturn>(method: String, arguments: Any...) async throws -> TReturn {
+        let (invocationId, tcs) = await invocationHandler.create()
+        invocationBinder.registerInvocation(invocationId: invocationId, types: TReturn.self)
+        defer {invocationBinder.removeInvocation(invocationId: invocationId)}
+        let invocationMessage = InvocationMessage(target: method, arguments: AnyEncodableArray(arguments), streamIds: nil, headers: nil, invocationId: invocationId)
+        let data = try hubProtocol.writeMessage(message: invocationMessage)
+        try await sendMessageInternal(data)
+        return try await tcs.task().result.value as! TReturn
```

Comment:
Looks the value might be `nil` when the customer invoke this method on `void` hub method. Then the type assertion is a fatal error.